packages/strapi-design-system/src/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { MutableRefObject } from 'react';
+
+import { TextareaInput, TextareaInputBoxProps } from './TextareaInput';
+import { Field, FieldLabel, FieldHint, FieldError, FieldProps } from '../Field';
+import { Flex } from '../Flex';
+import { useId } from '../hooks/useId';
+
+export interface TextareaProps extends TextareaInputBoxProps, Pick<FieldProps, 'hint' | 'error'> {
+  children: string;
```

Comment:
just the fact that we use `children` to alias `value`, why wouldn't we just use `value`? which I guess we can technically because of how we spread the props, maybe we should add a deprecation comment 🤔

Reply:
It probably comes from the fact, that in HTML the value is children: `<textarea>content</textarea>`, which is not true in React land. I'll add a deprecation notice - makes sense to me.